examples/cpp/CMakeLists.txt.cpp_example.in

```diff
@@ -25,7 +25,7 @@ if(CMAKE_CXX_COMPILER_ID STREQUAL "SunPro")
     set(CMAKE_EXE_LINKER_FLAGS "${CMAKE_EXE_LINKER_FLAGS} -m64")
 endif()
 
-find_package(@PRODUCT_NAME_NOSPACE@ REQUIRED COMPONENTS cpp)
+find_package(GeodeNative REQUIRED COMPONENTS cpp)
```

Comment:
Don't we want to have the FindGeodeNative.cmake file renamed to match the product name?

Reply:
https://github.com/apache/geode-native/blob/develop/examples/cmake/FindGeodeNative.cmake.in#L104

This line was failing, so we could replace the above line with `@PRODUCT_NAME_NOSPACE@` if that seems more resonable

Reply:
I agree with Jake.  Furthermore I'm not sure this change will work because our file generation will rename the Find script to the product name.  Check out https://github.com/apache/geode-native/blob/develop/examples/CMakeLists.txt and the line below.

```
configure_file(${CMAKE_CURRENT_SOURCE_DIR}/cmake/FindGeodeNative.cmake.in ${CMAKE_CURRENT_BINARY_DIR}/cmake/Find${PRODUCT_NAME_NOSPACE}.cmake @ONLY)
```

Reply:
Ok after trying on a different machine seems something is still wonky.  Gonna close for now and look after some eye rest and food.

@austxcodemonkey